platform/platform-impl/src/com/intellij/openapi/editor/impl/EditorComponentImpl.java

```diff
@@ -1192,6 +1199,8 @@ public void dispose() {
     // ---- Implements CaretListener ----
 
     private int myCaretPos;
+    private int previousCaretpos;
```

Comment:
Nitpick: we're usually adding `my` prefix for instance variables: https://plugins.jetbrains.com/docs/intellij/intellij-coding-guidelines.html#code-formatting. Also please capitalize `p` in `pos` as it's a separate word.
```suggestion
    private int myPreviousCaretPos;
```